docs/manual/introduction/How-to-update-things.html

```diff
@@ -113,6 +113,20 @@ <h3>[page:BufferGeometry]</h3>
 				<code>
 line.geometry.attributes.position.needsUpdate = true; // required after the first render
 				</code>
+					
+				<p>
+					If you change the position data values after the initial render, you should also call
+					`.computeBoundingSphere()` on the object to recalculate it's bounding sphere. 
+					The bounding sphere is used in calculations such as raycasting and frustrum culling.
+					If this method id not called after vertices are changed, you are likely to run into
+					strange side effects due to Three.js failing to properly identify the correct boundary
+				 	of your object. This may cause bugs succh as an object that doesn't properly
+					recieve raycasts on the entire object, or is frustrum culled too
+					early (such as when close to the edge of the screen).
```

Comment:
This is not a tutorial, so just the first sentence is all that is necessary.

@Mugen87 Maybe there are better places to mention this...

Reply:
Um, I thought the pages in the manual section (not the API reference) have actually tutorial/guide character. I think it's the right place for it.

Reply:
I do not think we need to emphasize bugs that can result from logic errors on the part of the user.

The most accurate statement is something like:

>If you change the position data values after the initial render, you may need to call `.computeBoundingSphere()` to recalculate the geometry's bounding sphere.

Reply:
Sounds good 👍

Reply:
Thanks for the suggestions! Updated accordingly.